packages/create-sitecore-jss/src/templates/vue/src/temp/GraphQLFragmentTypes.json

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
This file needs `JssVueWeb` replaced with `<%- helper.getAppPrefix(appPrefix, appName, false) %>`